dev-vcs/git/git-2.17.0.recipe

```diff
@@ -97,6 +97,7 @@ REQUIRES_email="
 	"
 REQUIRES_svn="
 	haiku
+#	alien_svn
```

Comment:
@pulkomandy, may I merge this PR with the `alien_svn` commented out?
(I have marked git 2.17.0 as untested on all architectures, althought it there are no regressions compared to 2.16.3 and 2.16.2.)
If I uncomment this line then I cannot test the recipe, as explained in https://github.com/haikuports/haikuporter/issues/151#issuecomment-377724701